app/assets/stylesheets/lux/_search_form.scss

```diff
@@ -1,31 +1,69 @@
 // bootstrap does flex-grow: 1 by default which makes the
 // text input roughly the same size as the select in many cases
+form.search-query-form {
+  width: 100%;
+  position: relative;
+  top: 2px;
+}
+
 .input-group {
   &.col-md-12 {
     padding-right: 0;
   }
-  > .form-control,
-  > .custom-select {
-    width: fit-content;
-  }
   > .custom-select {
     &.search-field {
-      flex-grow: 1;
-      max-width: fit-content;
+      height: $search-form-height;
+      max-width: $search-form-select-width;
+      margin-right: $search-form-margin-x;
+      padding: 0 $search-form-select-padding-left 0 $search-form-select-padding-right;
+
+      background: url("https://libraries.emory.edu/assets/icons/php/icon.php?id=material-keyboard_arrow_down&color=5387F7&size[width]=20&size[height]=20") no-repeat right 0.5rem center;
+      border-radius: $border-radius !important;
+      -moz-box-shadow:    inset 1px 2px 4px rgba(182, 189, 204, 0.5);
+      -webkit-box-shadow: inset 1px 2px 4px rgba(182, 189, 204, 0.5);
+      box-shadow:         inset 1px 2px 4px rgba(182, 189, 204, 0.5);
     }
   }
   > .form-control {
     &.search-q {
-      flex-grow: 3;
+      height: $search-form-height;
+      margin-right: $search-form-margin-x;
+      padding: $search-form-q-padding-y $search-form-q-padding-x;
+
+      border-radius: $border-radius-xs !important;
+      -moz-box-shadow:    inset 1px 2px 4px rgba(182, 189, 204, 0.5);
```

Comment:
Have you tried making the box-shadow more bootstrap-y? e.g. the final color could be one of our color variables, rather than rgba? See https://getbootstrap.com/docs/4.4/utilities/shadows/ for documentation

Reply:
As far as I can tell, there's no way to make an inset box-shadow in Bootstrap. Because the box-shadow also has a transparency value, we can't use our normal hex values. (It is possible to define 8-digit hex values to include transparency, but that is not well supported among browsers.) I did go ahead and define a new color `$base-slate-transparent` and variable `$search-form-box-shadow` to simplify things a bit.

Reply:
Search the bootstrap gem code for `btn-active-box-shadow` for an example of how to do this.